.github/workflows/scripts/build_nix.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/sh
 
-# fetch third_party libraries
-git submodule init && git submodule update
```

Comment:
I think we can leave this here, as we may want to refer to this script for users to try

Reply:
Build instructions are specified and explained in README, I think users shouldn't refer to workflows scripts (both build and release) since they are related to GitHub Actions environment.